akka-javasdk/src/main/scala/akka/javasdk/impl/AnySupport.scala

```diff
@@ -200,6 +203,97 @@ private[akka] object AnySupport {
     }
 
   def extractBytes(bytes: ByteString): ByteString = bytesToPrimitive(BytesPrimitive, bytes)
+
+  def toScalaPbAny(bytesPayload: BytesPayload): ScalaPbAny = {
```

Comment:
Need this for the commands that come from the fixed component endpoint (grpc)

Reply:
yes, we receive BytesPayload, but with proto contentType so it's not handled by JsonSerializer
these BytesPayload<->PbAny conversions are temporary